barretenberg/acir_tests/regenerate_verify_honk_proof_inputs.sh

```diff
@@ -19,9 +19,9 @@ export BRANCH
 # the program for which a proof will be recursively verified
 PROGRAM=assert_statement
 # the program containing the recursive verifier
-RECURSIVE_PROGRAM=verify_honk_proof
+RECURSIVE_PROGRAM=verify_rollup_honk_proof
```

Comment:
revert back to verify_honk_proof I guess or update so that this is a parameter of the script

Reply:
just regenerate both